DLCQuestipelago/DLCUnlockPatch/GrindstoneUnlockPackPatch.cs

```diff
@@ -236,4 +257,10 @@ private static void GetEnergyLinkJoulesAmount(ArchipelagoSession session, Action
             }
         }
     }
+
+    class EnergyLinkSetPacket : SetPacket

```

Comment:
Could this go in its own class? Maybe in the Archipelago folder where other server interaction stuff happen?

Reply:
could be yes, just felt silly for a class of 4 lines but sure ill move it

Reply:
Moved